docs/client-side-cases-by-proxy.md

```diff
@@ -9,11 +9,12 @@ However, since we have simplified the client-side logic as much as possible, thi
 so that the users do not need to choose the fields on projection. 
 This will also have a bad effect, we can only use all fields of the projection on the returned structure by default. 
 Moreover, we also need set to default the depth of nested queries so that the proxy can end smoothly. 
-All in all, the scenarios used are limited. Currently `.. on` and `alias` are not supported.
+All in all, the scenarios used are limited. Currently, `.. on` and `alias` not be supported.
```

Comment:
Why this change? *"X and Y are not supported"* was grammatically correct, *"X and Y not be supported"* is not correct.
I agree with the added comma, but the verb needs to stay conjugated.

Reply:
I want to say that this is passive

Reply:
But this *is* the passive voice. The active voice would be *"we don't support X"*. The passive voice is *"X is not supported"*. Using "be" here is just incorrect.

Reply:
ok,thanks.